src/pages/clubs/EventCreate.jsx

```diff
@@ -1,11 +1,16 @@
+/* eslint-disable react/no-unescaped-entities */
+/* eslint-disable no-unused-vars */
```

Comment:
Hi there, instead of disabling the rules for the file, won't it be better if we could follow the same ? 

Like removing unused vars, or making sure to use `&apos;` instead of `'` ?